blog/2025/pcsx2-2.6/index.mdx

```diff
@@ -0,0 +1,284 @@
+
+<PCSX2PRLink
+  authors="TheLastRar"
+  prNums="13466,12864,13360,13555,13557,13577,12864"
+>
+  ### Improved Display Surface (And Wayland!)
+</PCSX2PRLink>
+
+We have enabled Wayland support by default since PCSX2 2.4.0 release and in this cycle, TheLastRar has yet again spearheaded the work to improve Wayland support even further. This time we're cutting out the crufts of technical debts and implement it the way it (Qt) was intended.
+
+For more technical details, previously the way we render into "surfaces" was to just rendering it to the widget directly. While this was the canonical method for older versions of Qt, the Qt maintainers mentioned that this was not recommended and a newer method was better suited. This has caused some issues especially with Wayland.
+
+Instead, this new method now uses a Qt Window instead of a Qt Widget, avoiding issues inherited by the previous method and allowing for the removal of a number of Wayland specific workarounds.
+
+#### Simplify how we bundle Wayland plugins ([#12864](https://github.com/PCSX2/pcsx2/pull/12864))
+
+`linuxdeploy` doesn't correctly deploy some Wayland dependencies with the Wayland client modules.
+
+Previously, we manually copied these dependencies into the AppImage, however, it was discovered that specifying the Wayland compositor module will copy the needed client dependencies.
+
+This allowed us to simplify our AppImage build script.
+
+<PCSX2PRLink authors="Gonzalosilvalde" prNums="12653">
+  ### New Gamma Adjustment Option
+</PCSX2PRLink>
+
+Gonzalosilvalde has worked on adding gamma adjustment to shade boost within the post processing options. Adjusting the gamma allows you to intensify the shadows while keeping the overall color balance intact.
+
+<Image src={require("./img/gamma.webp").default} />
+
+<PCSX2PRLink authors="SternXD,flavionm" prNums="12953,12698">
+  ### Adjustable Position for Various In-Game Overlays
+</PCSX2PRLink>
+
+SternXD and flavionm has worked on a feature where you can adjust the positions of all overlays inside PCSX2 (such as achievements and OSD notifications)
+
+<Image src={require("./img/Achievement_Position.webp").default} />
+
+<PCSX2PRLink authors="Haisom" prNums="12640">
+  ### New Hotkey to Swap Memory Cards
+</PCSX2PRLink>
+
+Allow users to swap Memory Cards slot on demand using a hotkey (Unbound by default).
+
+To quote Haisom:
+
+> This is really useful on shared machines, specially with kids around (Forget kids accidentally overwriting your save games with over 100 hours of gameplay!).
+
+<PCSX2PRLink authors="fthomys" prNums="12934">
+  ### Organize Game Screenshot by Name in Folders
+</PCSX2PRLink>
+
+Have you ever been in a situation where your screenshot folder turn into a disorganized mess of game capture and snapshots? especially when you use the feature a lot.
```

Comment:
Plural "game captures"

Capitalize "Especially". The sentence does feel a bit odd, it might be better to encapsulate the thought into the previous sentence instead. Something like:

"Have you ever been in a situation where your snaps folder turns into a disorganized mess of recordings and screenshots after you have used these features for a while?"